src/browser/index.html

```diff
@@ -131,6 +131,72 @@ <h3 data-i18n="settings.category.general">
                         </div>
                     </div>
 
+                    <div
+                        class="settings_category"
+                        id="software-update"
+                        data-restriction-msstore
+                        data-restriction-os="win32"
+                    >
+                        <div class="category__header">
+                            <h3 data-i18n="settings.category.softwareUpdate">
+                                Software Update
+                            </h3>
+                        </div>
+
+                        <div class="category__content">
+                            <div class="settings_setting">
+                                <div class="setting_main">
+                                    <label data-for="config_autoUpdates">
+                                        Automatic Updates
+                                    </label>
+                                    <label class="cfgSwitch cfg_loading">
+                                        <input
+                                            type="checkbox"
+                                            id="config_autoUpdates"
+                                        />
+                                        <i></i>
```

Comment:
Is this element serving any purpose? If not, please remove it.

Reply:
Yes, it's needed for the switch style.